summoner-cli/src/Summoner/Template/Cabal.hs

```diff
@@ -80,139 +75,125 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
         settingsTestedVersions
 
     sourceRepository :: Text
-    sourceRepository =
-        [text|
-        $endLine
-        source-repository head
-          type:                git
-          location:            ${githubUrl}.git
-        |]
+    sourceRepository = unlines
+        [ ""
+        , "source-repository head"
+        , "  type:                git"
+        , "  location:            " <> githubUrl <> ".git"
+        ]
 
     commonStanza :: Text
-    commonStanza =
-        [text|
-        $endLine
-        common common-options
-          build-depends:       base $baseBounds
-          $customPrelude
-          $ghcOptions
-
-          default-language:    Haskell2010
-    |] <> defaultExtensions
+    commonStanza = unlines $
+        [ ""
+        , "common common-options"
+        , "  build-depends:       base " <> baseBounds
+        ]
+        <> customPrelude
+        <> ("" : map ("  " <>) ghcOptions)
```

Comment:
Can use `indent` here as well probably. Or maybe can add 2 spaces directly to `ghc-options`?
```suggestion
        <> ("" : map (indent 2) ghcOptions)
```